internal/player/player.go

```
@@ -99,36 +129,65 @@ func buildPlayCommand(url string, ipcPath string) (*exec.Cmd, string, error) {
+		}
+		return trimmed
+	}
+}
+
 func fetchMetadata(url string) (Metadata, error) {
 	ytbdlpPath, err := exec.LookPath("yt-dlp")
 	if err != nil {
-		return Metadata{}, fmt.Errorf("yt-dlp not found. Please install it and try again.")
+		return Metadata{}, fmt.Errorf("yt-dlp not found. Please install it and try again")
 	}
 	cmd := exec.Command(ytbdlpPath, "--no-playlist", "--print", "title,duration", url)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)